### DISCUSSION TITLE
请新增一下插件的总开关功能。没有找到停用插件的设置

### DISCUSSION THREAD

**Discussion by 2275273** [immortalwrt/homeproxy#110] [Q&A]
好像插件没有停用的功能。有时候测试一下其他插件需要先关闭一下这个插件，没有找到可以设置的地方。

**Comment by tanst** [comment]
虽然我知道怎么停用插件，但还是觉得作为一个 luci app 应该有个总开关。

**默认出站--禁用**

**Reply by xidongting** [reply]
禁用后，插件添加的nftables链没有被清掉

**Comment by boa-z** [comment]
同求，这个功能有添加的计划吗

**Comment by MkQtS** [comment]
luci: 系统 - 启动项 - homeproxy - 停止/禁用

或者命令行

```shell
/etc/init.d/homeproxy stop
/etc/init.d/homeproxy disable
```

**Comment by xiaofami** [comment]
`service homeproxy stop` 就可以了

<img width="919" height="295" alt="image" src="https://github.com/user-attachments/assets/a31350f0-cbb3-4874-bac3-a468650316c1" />
